CxScan/CxScanV20/task.json

```diff
@@ -170,6 +170,16 @@
       "groupName": "scanSettings",
       "visibleRule": "enableSastScan = true"
     },
+    {
+      "name": "overrideProjectSettings",
+      "type": "boolean",
+      "label": "Override Project Settings",
+      "required": false,
+      "defaultValue": "false",
+      "helpMarkDown": "Specifies whether to overwrite project settings. If set to false or empty, overwrite is determined by configuration value 'OverrideProjectSetting' in dbo.CxComponentConfiguration",
```

Comment:
"Select if project settings like Preset, EngineConfigurationID are to be updated in the CxSAST project. For new project, settings will be always saved on the CxSAST project. "